verible/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+sudo apt update
+sudo apt install -y gcc-8 g++-8
+
+export CC=gcc-8
+export CXX=g++-8
```

Comment:
Installing gcc-8 to get c++17 features - this method is copied from the Slang build script.